en/Obsidian Publish/Introduction to Obsidian Publish.md

```diff
@@ -0,0 +1,16 @@
+---
+alias: [Obsidian Publish]
+---
+
+Obsidian Publish is a cloud-based service where you can publish your notes to the world without leaving your favorite text editor.
+
+Select the notes you want to share with the world, press 'Publish', and we'll post it to publish.obsidian.md/your-site.
```

Comment:
wondering whether `'Publish'` should be `"Publish"` (English) or `**Publish**` (UI instruction convention as in the Sync PR). What do you think?

Reply:
Good catch! I wanted to describe the workflow at a high-level, so I'm not referring to a specific UI element here, but rather the action of publishing something. I switched to double-quotes.